src/Core/Scheduler/Providers/3dTiles_Provider.js

```diff
@@ -99,7 +99,7 @@ function getBox(volume, inverseTileTransform) {
 
 const rePosition = new RegExp('gl_Position.*(?![^]*gl_Position)');
 const reMain = new RegExp('[^\\w]*main[^\\w]*(void)?[^\\w]*{');
-function patchMaterial(material) {
+export function patchMaterial(material) {
```

Comment:
Nope: this PR exports a previously private function so you must either split it to a new file or nename it here.